src/__test__/components/data-exploration/differential-expression-tool/DiffExprResults.test.jsx

```diff
@@ -196,6 +196,22 @@ describe('DiffExprResults', () => {
     expect(table.getElement().props.data.length).toEqual(5);
   });
 
+  it('is sorted by descending logFC by default', () => {
+    const component = mount(
+      <Provider store={withResultStore}>
+        <DiffExprResults experimentId={experimentId} onGoBack={jest.fn()} width={100} height={200} />
+      </Provider>,
+    );
+
+    const table = component.find('Table');
+    expect(table.getElement().props.columns[1].sortOrder).toEqual(null);
+    expect(table.getElement().props.columns[2].sortOrder).toEqual('descend');
+    expect(table.getElement().props.columns[3].sortOrder).toEqual(null);
+    expect(table.getElement().props.columns[4].sortOrder).toEqual(null);
+    expect(table.getElement().props.columns[5].sortOrder).toEqual(null);
+    expect(table.getElement().props.columns[6].sortOrder).toEqual(null);
```

Comment:
Thanks for adding this. Can you add a test to check the value is properly sorted well? (not only the settings)